docs/docs/resources/variables/config_env_vars.md

```diff
@@ -1,4 +1,4 @@
-These variables are a lower priority alternative to the settings in `config.yaml`. Variables marked as required can instead be set in the global config.
+These variables take precedence over the settings in `config.yaml`. Variables marked as required can instead be set in the global config.
```

Comment:
How about "these variables _always_ take precedence"? (over CLI flags, config.yaml, dotenv)

Reply:
It seems to me that this section specifically talks about setting values that can otherwise be specified in config.yaml. I'll add a similar wording to the CLI section below. Does that work for you?